dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -382,6 +383,15 @@ export const LiveLayout = ({ CAPI, NAV, format, palette }: Props) => {
 	const { branding } = CAPI.commercialProperties[CAPI.editionId];
 	return (
 		<>
+			{format.design === ArticleDesign.LiveBlog && (
```

Comment:
This structure of wrapping Liveness within an Island feels very 'natural' and in-keeping with the model for components requiring hydration. Given you're familiar with dcr's Island pattern, I think it's obvious what we're achieving by doing this.

Could you help me understand: the `deferUntil="idle"` implies we will only do the hydration when the browser is [idle](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback), meaning we're not consuming any compute until the page has fully loaded?

Reply:
I think the actual implementation of when idle gets fired might vary browser to browser but in general, yes, it offers a way to defer code execution to a 'lower priority' position. We only really need this code to run after a few seconds so it's much less important than code related to the main rendering thread.

From a technical point of view, the `deferUntil` prop here is wrapping the code in a listener which only gets fired once the browser declares that it is 'idle'